### ISSUE TITLE
Crowdstrike results in wrong places

### ISSUE THREAD

**Issue by samsson** [mitre-attack/attack-evals#6]
"15.A.1 Input Capture (T1056)" contains results from "15.B.1 Credentials in Files (T1081) " and vice versa

**Comment by jburns12** [comment]
Thanks for letting us know about this as well, @samsson. We'll close the issue once this fix is merged into the site.

**Comment by ghost** [comment]
FIXED: Step 15.A.1 (Input Capture) and Step 15.B.1 (Credentials in Files) were in the wrong order. They have been switched to reflect the correct order of execution.
